openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v140/MigrationUtil.java

```diff
@@ -0,0 +1,127 @@
+package org.openmetadata.service.migration.utils.v140;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonObject;
+import javax.json.JsonString;
+import javax.json.JsonValue;
+import lombok.extern.slf4j.Slf4j;
+import org.jdbi.v3.core.Handle;
+import org.openmetadata.schema.api.services.CreateDatabaseService;
+import org.openmetadata.schema.entity.data.Table;
+import org.openmetadata.schema.type.PartitionColumnDetails;
+import org.openmetadata.schema.type.PartitionIntervalTypes;
+import org.openmetadata.schema.type.TablePartition;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.openmetadata.service.resources.databases.DatasourceConfig;
+import org.openmetadata.service.util.JsonUtils;
+import org.postgresql.util.PGobject;
+
+@Slf4j
+public class MigrationUtil {
+  private static final String MYSQL_QUERY_TABLES_WITH_PARTITION =
+      "SELECT json "
+          + "FROM table_entity "
+          + "WHERE JSON_EXTRACT(json, '$.tablePartition') IS NOT NULL";
+
+  private static final String POSTGRES_QUERY_TABLES_WITH_PARTITION =
+      "SELECT json " + "FROM table_entity " + "WHERE json->'tablePartition' IS NOT NULL";
```

Comment:
I guess this could be a single string 🤷 spotless something does weird things

Reply:
Haha, yep, I think it is because I had it across 3 lines and spotless just brings it back into one